ios/src/ImageCropPicker.m

```diff
@@ -424,6 +556,9 @@ - (void) getVideoAsset:(PHAsset*)forAsset completion:(void (^)(NSDictionary* ima
             [self.loadingLabel setText:@"Downloading from iCloud..."];
         });
     };
+    
+    
+    
     // Create temp file
     NSString *filePath = [[[self getTmpDirectory] stringByAppendingString:[[NSUUID UUID] UUIDString]] stringByAppendingString:@".mp4"];
```

Comment:
.mp4 -> can you unify the extension extraction logic so we won't have 2 logic hanging around?
same with mime type - as it is used multiple times.